client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -120,6 +101,7 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 }
 
 export default connect( ( state, { siteId, steps } ) => ( {
+	isConnected: isJetpackSite( state, siteId ), // Will only return true if it's connected to WP.com
```

Comment:
We don't need this for other reason than for determining which `Todo` component to use, so that could be moved there as well.